gc_registry/certificate/validation.py

```diff
@@ -55,18 +55,17 @@ def validate_granular_certificate_bundle(
     device_mw = device_w / W_IN_MW
     device_max_watts_hours = device_mw_capacity_to_wh_max(device_mw, hours)
 
-    if not device_max_certificate_id:
-        device_max_certificate_id = 0
+    CAPACITY_MARGIN = 1.1  # TODO: Review what the margin should be - Punped storage seemed to break this validation
```

Comment:
Maybe a settings entry instead? Wonder if there's going to be a lot of validation-specific constants it would be worth having a separate config for those.

Reply:
I think we can just specify these defaults directly in the settings.py for clarity instead of `.env` - what d'you reckon?

Reply:
That's a good idea, they'll be more visible there than in the .env. Maybe they can be overridden if they're in the .env but kept on the default if they're not present, to not overload the user with options

Reply:
Hmmm - yeah didn't think about them being over-written from the .env. Could potentially hard-code after the settings init